libs/lb/lb-rs/Cargo.toml

```diff
@@ -16,6 +16,7 @@ bench = false
 [features]
 default = []
 no-network = ["db-rs/clone"]
+serve = []
```

Comment:
imo this is better of being configured by `core_config.rs` rather than `Cargo.toml`